tests/testthat/test-variable-summary.R

```diff
@@ -1,91 +1,97 @@
-    })
-
-    test_that("unsupported table methods", {
-        expect_error(table(gen, 1:5),
-            "Cannot currently tabulate Crunch variables with non-Crunch vectors")
-        expect_error(table(1:5, gen),
-            "Cannot currently tabulate Crunch variables with non-Crunch vectors")
-        expect_error(table(), "nothing to tabulate")
-    })
-
-    test_that("table makes a cube request", {
-        expect_GET(table(gen), "https://app.crunch.io/api/datasets/1/cube/")
-    })
-
-    test_that("unsupported aggregation methods", {
-        expect_error(mean(ds$textVar),
-            paste(dQuote("mean"), "is not defined for TextVariable"))
-        expect_error(sd(ds$textVar),
-            paste(dQuote("sd"), "is not defined for TextVariable"))
-        expect_error(median(ds$textVar),
-            paste(dQuote("median"), "is not defined for TextVariable"))
-        expect_error(min(ds$textVar),
-            paste(dQuote("min"), "is not defined for TextVariable"))
-        expect_error(max(ds$textVar),
-            paste(dQuote("max"), "is not defined for TextVariable"))
-    })
-
-    test_that("Summary method for numeric", {
-        expect_is(summary(ds$birthyr), "NumericVariableSummary")
-    })
+  ds <- loadDataset("test ds")
+  gen <- ds$gender
+  
+  test_that("table 'method' dispatch", {
+    expect_identical(table(1:5), base::table(1:5))
+    expect_identical(table(useNA="ifany", 1:5),
+                     base::table(useNA="ifany", 1:5))
+    expect_identical(table(useNA="ifany", c(NA, 1:5)),
+                     base::table(useNA="ifany", c(NA, 1:5)))
+  })
+  
+  test_that("unsupported table methods", {
+    expect_error(table(gen, 1:5),
+                 "Cannot currently tabulate Crunch variables with non-Crunch vectors")
+    expect_error(table(1:5, gen),
+                 "Cannot currently tabulate Crunch variables with non-Crunch vectors")
+    expect_error(table(), "nothing to tabulate")
+  })
+  
+  test_that("table makes a cube request", {
+    expect_GET(table(gen), "https://app.crunch.io/api/datasets/1/cube/")
+  })
+  
+  test_that("unsupported aggregation methods", {
+    expect_error(mean(ds$textVar),
+                 paste(dQuote("mean"), "is not defined for TextVariable"))
+    expect_error(sd(ds$textVar),
+                 paste(dQuote("sd"), "is not defined for TextVariable"))
+    expect_error(median(ds$textVar),
+                 paste(dQuote("median"), "is not defined for TextVariable"))
+    expect_error(min(ds$textVar),
+                 paste(dQuote("min"), "is not defined for TextVariable"))
+    expect_error(max(ds$textVar),
+                 paste(dQuote("max"), "is not defined for TextVariable"))
+  })
+  
+  test_that("Summary method for numeric", {
+    expect_is(summary(ds$birthyr), "NumericVariableSummary")
+  })
+  test_that("max", {
+    expect_equal(max(ds$birthyr), 1.6662)
+  })
+  test_that("min", {
+    expect_equal(min(ds$birthyr), -1.4967)
+  })
```

Comment:
Is this just a whitespace removal here, or is there something else that's changed? Github's view is not making it easy for me to tell: it looks like just whitespace to me, but I might be missing something.